packages/reporters/dev-server/src/ServerReporter.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import {Reporter} from '@parcel/plugin';
+import Server from './Server';
+
+const DEFAULT_CACHE_DIR = '.parcel-cache';
+
+let servers: Map<number, Server> = new Map();
```

Comment:
Added support for running multiple servers, was originally a workaround for the tests but we should probably keep this as there are probably use cases where people would wanna open multiple instances of parcel.
Might even be useful for a centralised dev server of some kind with multiple devs/projects?